contracts/PrecedenceCampaignArbitrable.sol

```diff
@@ -70,6 +75,10 @@ contract PrecedenceCampaignArbitrable is IArbitrable {
         treasury.withdraw(_token, _to, _amount);
     }
 
+    function recoverFunds(address _token, address payable _to, uint256 _amount) external only(owner) {
+        require(ERC20(_token).transfer(_to, _amount), ERROR_RECOVER_FUNDS_FAILED);
```

Comment:
Wouldn't it be better to use `safeTransfer` here, just in case?
See [here](https://github.com/aragonone/precedence-campaign-arbitrable/pull/3/files) where I used exactly the same as we use for our Court contracts.

Reply:
Yup, I missed that, fixed in https://github.com/aragonone/precedence-campaign-arbitrable/commit/4cc4d7cfbc672db588be1d83d748a08f28e886cf